tools/pytools/tmp/.gitignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
the working dir of `pydoctor` under poetry should be subdir of the `pytools_path`